codecs.go

```diff
@@ -3,8 +3,15 @@ package rsmt2d
 import "fmt"
 
 const (
+	// Leopard is a codec that was originally implemented in the C++ library
+	// https://github.com/catid/leopard. rsmt2d uses a Go port of the C++
+	// library in https://github.com/klauspost/reedsolomon. The Leopard codec
+	// uses 8-bit leopard for shards less than or equal to 256. The Leopard
+	// codec uses 16-bit leopard for shards greater than 256.
 	Leopard = "Leopard"
-	RSGF8   = "RSFG8"
+	// RSGF8 stands for Reed-Solomon Galois Field 8-bit mode. This codec uses
```

Comment:
> RSGF8 stands for Reed-Solomon Galois Field 8-bit mode.

This was my best guess because I couldn't find a reference that explains this name.